src/Migrate.php

```diff
@@ -1429,20 +1432,20 @@ private function grantUsageToOldTable(
             sprintf(
                 'grant usage on database %s to role %s;',
                 Helper::quoteIdentifier($database . '_OLD'),
-                $role
+                Helper::quoteIdentifier($role)
             ),
             sprintf(
-                'grant usage on schema %s in database %s to role %s;',
-                Helper::quoteIdentifier($schema),
+                'grant usage on schema %s.%s to role %s;',
                 Helper::quoteIdentifier($database . '_OLD'),
-                $role
+                Helper::quoteIdentifier($schema),
+                Helper::quoteIdentifier($role)
             ),
             sprintf(
-                'grant usage on table %s in schema %s.%s to role %s;',
-                Helper::quoteIdentifier($table),
+                'grant select on table %s.%s.%s to role %s;',
```

Comment:
j, I thought that usage maybe wouldn't be enough

Reply:
why not?